gips/inventory/orm/__init__.py

```diff
@@ -34,7 +34,7 @@ def setup():
     if setup_complete:
         return
     if use_orm():
-        busted_drivers = ('sar', 'sarannual', 'modaod', 'merra', 'daymet', 'cdl')
+        busted_drivers = ('sar', 'sarannual', 'modaod', 'daymet', 'cdl')
```

Comment:
To my shame I didn't document why I blacklisted merra in the first place, but would you mind describing how you know merra is safe?  Like what testing you may've done, etc?